lib/newman_conway.py

```diff
@@ -3,4 +3,18 @@ def newman_conway(num):
         Time Complexity: ?
         Space Complexity: ?
     """
-    pass
+    if num <= 0:
+        raise ValueError(0)
+    else: 
+        arr = [0, 1, 1]
+
+    if num == 1 or num == 2:
```

Comment:
```suggestion
    if num == 1:
```

Great work on this overall, Shayla! The one bug keeping the tests from passing is stripping out the `or num == 2` due to the fact that the function is expected to return "1 1" for an input of 2. This may have been poorly spelled out in the description, my apologies for the confusion!